modules/test/playwright/tests/portal-workflow-kaleo-designer-web/dependencies/sample-client-extension-workflow-definition.xml

```diff
@@ -83,7 +83,7 @@
 		</metadata>
 		<actions>
 			<action>
-				<name>timestamp</name>
+				<name>action1</name>
```

Comment:
Are you sure that the modifications in this xml file does not break other tests that have it as a dependency? Have you ran those?